internal/vds/vds_test.go

```diff
@@ -427,40 +427,46 @@ func TestFence(t *testing.T) {
 
 func TestFenceBorders(t *testing.T) {
 	testcases := []struct {
+		name              string
 		coordinate_system int
 		coordinates       [][]float32
 		interpolation     string
 		error             string
 	}{
 		{
+			name:              "coordinate 2 is just-out-of-upper-bound in direction 0",
 			coordinate_system: CoordinateSystemAnnotation,
-			coordinates:       [][]float32{{3, 10}, {6, 11.25}},
+			coordinates:       [][]float32{{5, 9.5001}, {6, 11.25}},
 			interpolation:     "nearest",
-			error:             "Coordinate (6.000000,11.250000) is out of boundaries in dimension 0.",
+			error:             "is out of boundaries in dimension 0.",
 		},
 		{
+			name:              "coordinate 1 is just-out-of-upper-bound in direction 1",
 			coordinate_system: CoordinateSystemAnnotation,
 			coordinates:       [][]float32{{5.5, 11.5}, {3, 10}},
 			interpolation:     "linear",
-			error:             "Coordinate (5.500000,11.500000) is out of boundaries in dimension 1.",
+			error:             "is out of boundaries in dimension 1.",
 		},
 		{
+			name:              "coordinate is long way out of upper-bound in both directions",
 			coordinate_system: CoordinateSystemCdp,
 			coordinates:       [][]float32{{700, 1200}},
 			interpolation:     "cubic",
-			error:             "Coordinate (700.000000,1200.000000) is out of boundaries in dimension 0.",
+			error:             "is out of boundaries in dimension 0.",
 		},
 		{
+			name:              "coordinate 2 is just-out-of-lower-bound in direction 1",
 			coordinate_system: CoordinateSystemAnnotation,
-			coordinates:       [][]float32{{3, 11}, {5, 10}, {1, 9.5}},
+			coordinates:       [][]float32{{0.0001, 11}, {5.9999, 10}, {0.0001, 9.5}},
```

Comment:
Is there any point in have multiple coordinates here? Why not keep it simple and clear by having a single coordinate?

Reply:
Again - slight not really important "randomness". For example, is the first wrong coordinate reported, or is the last one, or are all the ones?
(result was btw way more clear with full error message)